components/aws/sagemaker/common/_utils.py

```diff
@@ -517,7 +559,17 @@ def create_endpoint_config_request(args):
 
     return request
 
+def get_endpoint_config_name(client, args):
+    endpoint_config_name = args['endpoint_config_name']
```

Comment:
nit: better to pass endpoint_config_name directly after checking for its existance